install/tools/ipa-replica-conncheck

```diff
@@ -509,57 +507,64 @@ def main():
                         stderr = e
                     else:
                         for message in result['messages']:
-                            print_info(message['message'])
+                            root_logger.info(message['message'])
                         returncode = int(not result['result'])
                         stderr = ("ipa-replica-conncheck returned non-zero "
                                   "exit code")
                     finally:
                         if api.Backend.rpcclient.isconnected():
                             api.Backend.rpcclient.disconnect()
             except Exception:
-                print_info("Retrying using SSH...")
+                root_logger.info("Retrying using SSH...")
 
                 # Ticket 5812 Always qualify requests for admin
                 user = principal
                 ssh = SshExec(user, options.master)
 
-                print_info("Check SSH connection to remote master")
+                root_logger.info("Check SSH connection to remote master")
                 result = ssh('echo OK', verbose=True)
                 if result.returncode != 0:
-                    print('Could not SSH into remote host. Error output:')
-                    for line in result.error_output.splitlines():
-                        print('    %s' % line)
-                    raise RuntimeError('Could not SSH to remote host.')
+                    root_logger.debug(result.error_output)
```

Comment:
It is purely a debug information (~50-100 lines), so I don't see the reason why it should be displayed to the user. Since previously the function lacked any reasonable logging, I think it was decided to simply print this debug info out, as the rest of the messages.

Since this patch adds logging to the function, I this the proper place for this information is the log. If the command fails, the user is prompted to check the log where he can find this info.